winml/lib/Api.Image/inc/D3DDeviceCache.h

```diff
@@ -104,9 +106,8 @@ class D3DDeviceCache {
   winrt::com_ptr<ID3D12RootSignature> tensorize_root_signature_;
   winrt::com_ptr<ID3D12RootSignature> detensorize_root_signature_;
 
-  winrt::com_ptr<ID3D12PipelineState>
-    cached_pipeline_state[PipelineStateCacheType::kCount][PipelineStateCacheFormat::kCount]
-                         [PipelineStateCacheFormat::kCount][PipelineStateCacheOperation::kCount];
+  winrt::com_ptr<ID3D12PipelineState> cached_pipeline_state[to_underlying_integer(PipelineStateCacheType::kCount)][to_underlying_integer(PipelineStateCacheFormat::kCount)][to_underlying_integer(PipelineStateCacheFormat::kCount)]
```

Comment:
🤔 Hmm, was more readable before spread across lines.

In cases where clangformat is being dumb, you can use `// clang-format off` (not that this necessarily needs it, but I've encountered that frequently with tables).

Reply:
Yeah